dashboards/google-application-integration/data-processed.json

```diff
@@ -0,0 +1,220 @@
+{
+    "category": "CUSTOM",
+    "dashboardFilters": [
+      {
+        "filterType": "RESOURCE_LABEL",
+        "labelKey": "location",
+        "templateVariable": ""
+      },
+      {
+        "filterType": "RESOURCE_LABEL",
+        "labelKey": "integration",
+        "stringValue": "shubham-monitoring-integration",
```

Comment:
Omit this line

Reply:
Removed it, thanks for catching it.

Thanks for reviewing the CL.